src/connectors/aws_collect.py

```diff
@@ -662,6 +662,27 @@
         ('created_at', 'TIMESTAMP_NTZ'),
         ('updated_at', 'TIMESTAMP_NTZ'),
     ],
+    # https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-snapshots.html
```

Comment:
link directly to the output section of the page, as the others do — https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-snapshots.html#output